addons/base_vat/models/res_partner.py

```diff
@@ -74,6 +74,7 @@
     'sk': 'SK2022749619',
     'sm': 'SM24165',
     'tr': _('17291716060 (NIN) or 1729171602 (VKN)'),
+    'uy': _("'219999830019' (should be 12 digits)"),
```

Comment:
```suggestion
    'uy': _("219999830019 (12 digits)"),
```
And you'll need to re-export the pot please.